Rover/GCS_MAVLink_Rover.cpp

```diff
@@ -500,6 +500,13 @@ MAV_RESULT GCS_MAVLINK_Rover::handle_command_int_packet(const mavlink_command_in
 {
     switch (packet.command) {
 
+    case MAV_CMD_CONDITION_YAW:
+        if (rover.g2.motors.have_skid_steering()) {
```

Comment:
We probably should have a "can_pivot_turn()" here, but this is fine.  I don't think omnis should be returning "true" for have_skid_steering as there are many ways to do omnis that aren't skid steering

Not your problem to fix :-)